src/metrics/codesmells_metrics.nit

```diff
@@ -41,10 +41,14 @@ class CodeSmellsMetricsPhase
 		for mclass in mainmodule.flatten_mclass_hierarchy do
 			mclass_codesmell.collect(mclass.mclassdefs,self)
 		end
-		if toolcontext.opt_get_all.value then
-			mclass_codesmell.print_all
+		if toolcontext.opt_print_stats.value then
+			mclass_codesmell.print_global_stat(self)
 		else
-			mclass_codesmell.print_top(10)
+			if toolcontext.opt_get_all.value then
```

Comment:
~~~nit
if foo then
else if bash then
else
end
~~~

instead of
~~~nit
if foo then
else
  if bash then
  else
  end
end
~~~